core/src/test/java/org/elasticsearch/tribe/TribeIT.java

```diff
@@ -554,17 +553,17 @@ private void putCustomMetaData(InternalTestCluster cluster, final TestCustomMeta
     private static void updateMetaData(InternalTestCluster cluster, UnaryOperator<MetaData.Builder> addCustoms) {
         ClusterService clusterService = cluster.getInstance(ClusterService.class, cluster.getMasterName());
         final CountDownLatch latch = new CountDownLatch(1);
-        clusterService.submitStateUpdateTask("update customMetaData", new ClusterStateUpdateTask(Priority.IMMEDIATE) {
+        clusterService.submitStateUpdateTask("update customMetaData", new LocalClusterUpdateTask(Priority.IMMEDIATE) {
```

Comment:
why did you change this? it seems like it supposed to publish a new cluster state?